x-pack/plugins/security_solution/public/flyout/alerts/panels/right/tabs/test_ids.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const OVERVIEW_TAB_CONTENT_TEST_ID = 'securitySolutionAlertDetailsFlyoutOverviewTabContent';
```

Comment:
I like this pattern and ultra separation of concerns. I know people tend to keep items like this in `constants.ts` files. I'm not sure which is better, but just wanted to point it out

Reply:
yeah I've seen many `translations.ts` files but not as many `test_ids.ts` files. What I like about having them separated, is when we declare variable in Cypress (within the `screens` folder) we can import those  from the `test_ids` files. It avoids making typo mistakes